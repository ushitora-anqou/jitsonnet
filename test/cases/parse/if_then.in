if true then 1.0
