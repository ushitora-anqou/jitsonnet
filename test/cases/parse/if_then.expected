{ Syntax.v =
  (Syntax.If
     ({ Syntax.v = Syntax.True;
        loc =
        (Some { Syntax.startpos =
                { Syntax.fname = "test/cases/parse/if_then.in"; line = 1;
                  column = 3 };
                endpos =
                { Syntax.fname = "test/cases/parse/if_then.in"; line = 1;
                  column = 7 }
                })
        },
      { Syntax.v = (Syntax.Number 1.);
        loc =
        (Some { Syntax.startpos =
                { Syntax.fname = "test/cases/parse/if_then.in"; line = 1;
                  column = 13 };
                endpos =
                { Syntax.fname = "test/cases/parse/if_then.in"; line = 1;
                  column = 16 }
                })
        },
      None));
  loc =
  (Some { Syntax.startpos =
          { Syntax.fname = "test/cases/parse/if_then.in"; line = 1;
            column = 0 };
          endpos =
          { Syntax.fname = "test/cases/parse/if_then.in"; line = 1;
            column = 16 }
          })
  }